ModernWallet/Classes/ViewController/Transactions/TransactionsStep1ViewController.swift

```diff
@@ -108,9 +108,6 @@ extension TransactionsStep1ViewController: UISearchResultsUpdating {
                 fatalError("Unable to instatiate a SearchResultsViewController from the storyboard.")
         }
         
-        if let viewModel = viewModel {
```

Comment:
Here we reuse the view model in order to use the same data, which is already fetched.Could you please find a way to keep this logic I try to find another solution in order to fix LMW-359 ? Thanks.